datafusion/physical-expr/src/expressions/in_list.rs

```diff
@@ -52,16 +54,27 @@ use datafusion_expr::ColumnarValue;
 static OPTIMIZER_INSET_THRESHOLD: usize = 30;
 
 /// InList
-#[derive(Debug)]
 pub struct InListExpr {
     expr: Arc<dyn PhysicalExpr>,
     list: Vec<Arc<dyn PhysicalExpr>>,
     negated: bool,
     set: Option<InSet>,
+    input_schema: Schema,
```

Comment:
Why does this struct need `input_schema`? It seems to me it is not needed

Reply:
This is because `InListExpr::new()` method need the  schema.